go/host/enclave/guardian.go

```diff
@@ -626,55 +625,6 @@ func (g *Guardian) periodicRollupProduction() {
 	}
 }
 
-func (g *Guardian) periodicBundleSubmission() {
```

Comment:
did you mean to remove this?
Shouldn't matter, but it might cause issues when cherry-picking to main